backend/app/app.go

```diff
@@ -85,6 +86,7 @@ func Start(
 		kgsBufferSize,
 		kgsRPCConfig,
 		provider.TokenValidDuration(config.AuthTokenLifetime),
+		provider.FeedbackSlackWebHook(config.FeedbackSlackWebHook),
```

Comment:
provider.FeedbackSlackWebHook doesn't exist

Reply:
Accidentally checked it in.